dxvk.conf

```diff
@@ -339,6 +339,24 @@
 # environment variable is not set, otherwise it will be
 # ignored. The syntax is identical.
 
+# a valid value is : devinfo,fps,frametimes,pipelines,memory,version,api,compiler,
+
+#  here the different elements you can use 
```

Comment:
Capitalize the first letter

To make it consistent with the readme i'd probably change the wording to "options"
e.g "The following options are supported" or similar